README.md

```diff
@@ -111,7 +111,29 @@ $ faas-cli deploy -f ./samples.yml
 
 ### Managing secrets
 
-You can deploy secrets and configuration via environmental variables.
+You can deploy secrets and configuration via environmental variables in-line or via external files.
+
+> Note: external files take priority over in-line environmental variables. This allows you to specify a default and then have overrides within an external file.
```

Comment:
worth making the priority clear when the same envvar appears in multiple files, is it the first file loaded, the last etc?